uaa/src/test/java/org/cloudfoundry/identity/uaa/mock/providers/IdentityProviderEndpointsAliasMockMvcTests.java

```diff
@@ -327,85 +373,358 @@ protected UpdateBase(final boolean aliasFeatureEnabled) {
             void setUp() {
                 arrangeAliasFeatureEnabled(aliasFeatureEnabled);
             }
-        }
-
-        @Nested
```

Comment:
This is confusing. 

It looks like you didn't actually delete this, but that git is confused and believes so. It makes it harder to review. That's one of the many reasons I like smaller commits, it makes it easier on git and us.

Reply:
For this particular PR, it might be easier to just consider the new version of the test class and ignore the diffs git generated. The PDF I posted in the Slack channel might be helpful as well.

Reply:
But then I don't see what is being deleted. I can't validate that we are not removing test coverage. 

The commit says that it adds further tests, but I see lots of lines of code being deleted.